### DISCUSSION TITLE
made it work ty

### DISCUSSION THREAD

**Discussion by nerves2014** [terminatorul/NvStrapsReBar#46] [Q&A] [answered]

**Comment by UnidentifiedTag** [comment]
You should take this to the issues section found [here](https://github.com/terminatorul/NvStrapsReBar/issues)

**Comment by xCuri0** [comment]
why are you building there's no reason to @nerves2014

**Reply by nerves2014** [reply]
Wait what should i do then? I just tried to follow the vid found on yt xD

**Reply by nerves2014** [reply]
Btw i have ryzen 5 5600x rtx 2060
Mpg B550 gaming its not working on ami bios

**Reply by xCuri0** [reply]
@nerves2014 read the README, it's very simple to do on AM4 if you can figure out flashing the modded BIOS (i've heard its kinda hard on AM4)

Steps are basically these

1. Add NvStrapsRebar module
2. Patch BIOS to support Resizable BAR (but **do not** add ReBarDxe) if you have an older system without it (you can skip this because recent motherbord)
3. Flash BIOS and use NvStrapsRebar.exe to configure it

**Reply by nerves2014** [reply]
Copy copy i will try it rn i'll update if it works

**Reply by nerves2014** [reply]
I have two system 1 is r5 3600 gtx 1660ti b550m I'll try it on the r5 3600 first

**Reply by nerves2014** [reply, marked as answer]
i did this and it worked 
My motherboard supports 4G decoding, so the BIOS could only be updated using USB BIOS FlashBack.

I modified the BIOS file using UEFITool (https://github.com/xCuri0/ReBarUEFI/wiki/Adding-FFS-module)
Specified NvStrapsReBar.ffs
Saved the file and renamed it to the correct name for the BIOS. No need to use UEFIPatch!
Turned off the PC. Installed Bios via USB BIOS FlashBack
Launched NvStrapsReBar.exe. Pressed E, S, restarted the PC. According to the instructions.
4G decoding must be activated in the BIOS!
